pkg/providers/vsphere/network/network.go

```diff
@@ -661,16 +661,17 @@ func vpcSubnetPortToResult(
 
 	ipConfigs := []NetworkInterfaceIPConfig{}
 
-	// No DHCP supported in VPC at the moment.
-	ipAddress := subnetPort.Status.IPAddresses
+	// IPAddresses has CIDR format.
+	ipAddress := subnetPort.Status.NetworkInterfaceConfig.IPAddresses
```

Comment:
A little confusing to use a singular var name. We could get away w/o using an intermediate variable